src/vstart.sh

```diff
@@ -761,6 +757,20 @@ EOF
     done
 fi
 
+# Don't set max_mds until all the daemons are started, otherwise
+# the intended standbys might end up in active roles.
+fs=0
+for name in a b c d e f g h i j k l m n o p
+do
+    sleep 5  # wait for daemons to make it into FSMap
```

Comment:
This appears to be sleeping 5 seconds for each fs. I don't think that was meant to happen?